charts/kyverno/templates/background-controller/role.yaml

```diff
@@ -36,5 +36,13 @@ rules:
       - update
     resourceNames:
       - kyverno-background-controller
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
I see that we set `resourceNames` when grating access to configmaps in this role. Can this be done for secrets also, or are the names not known?

Reply:
Secrets names can be changed if the namespace is different. Kyverno internally watches the two managed secrets only.

Reply:
I added the secrets names via https://github.com/kyverno/kyverno/pull/8721, names in `config/install-latest-testing.yaml` are always the default.